PMurphy1940/HandPickt
# Notes section

# Description
The notes section displays all user notes. A user may click on any note to view a zoomed in version of that note. The zoomed view will allow for the editing and deleting of the note. Users may create a new note by clicking on the image of a postIt note in the NavBar
## Type of change


# Testing Instructions

While logged in, you may view saved notes by clicking on the ```Notes``` button on the dashboard, or from any view buy clicking on the clipboard icon in the NavBar. Notes will be displayed in two rows. Any notes that pertain to the current day of week will have a flashing ```Today!```  displayed on them. Clicking on any of the notes will take the user to a ```Details``` view of the note, affording ```Delete``` and ```Edit```(same as described in Add New below.)
In the notes view, the center icon in the NavBar will change to a PostIt note. Clicking the PostIt will take the user to the ```Add new note``` view. Adding a new note will allow you to save a comment(the note), change the randomly selected PostIt note image if you don't approve of the current one(there are 7 choices), and set the note to be reoccurring. If selected to reoccurring, a button will appear in the upper right corner of the user area. The button will open a modal and allow the user to select the days of the week on which the note will pertain. When any changes are made, a ```Save Note``` button will appear at the bottom of the view.
From the ```Dashboard``` view, if there are notes that pertain to the current day, a small, pulsing PostIt icon will appear in the corner of the Notes button.
# Checklist:
